index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Welcome to my portfolio. I'm a raccoon.</title>
+        <link rel="stylesheet" type="text/css" href="styles.css">
+        <link href="https://fonts.googleapis.com/css2?family=Fira+Mono:wght@400;700&family=Pangolin&family=Swanky+and+Moo+Moo&display=swap" rel="stylesheet"> 
+    </head>
+    <body>
+        <nav id="navbar">
+            <h1 class="site-title">Welcome to my portfolio v3.1</h1>
+            <ul class="navbar-links">
+                <a href="#welcome-section">Welcome</a>
+                <a href="#projects">Projects</a>
+                <a href="#about-contact">About</a>
+                <a href="#sitemap">Sitemap</a>
+            </ul>
+        </nav>
+        <main id="wrapper">
+            <section id="welcome-section">
+                <div class="centered-box">
+                    <h1 class="welcome-title">This is the welcome message</h1>
+                    <button class="centered-button">Go to portfolio</button>
```

Comment:
It's confusing to have a button that doesn't do anything. I think you can surround it with a link.